.github/workflows/ci.yml

```diff
@@ -132,12 +137,12 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-target-test-${{ matrix.rust }}-${{ hashFiles('**/Cargo.toml') }}
             ${{ runner.os }}-target-test-${{ matrix.rust }}-
+
       - run: cargo test --workspace --all-targets
+      - run: cargo test --workspace --all-targets --features docking
```

Comment:
There's a thing in the xtask to do the tests like CI does, can we mirror this change there? (It's possible now that I'm not making many changes to this stuff that code isn't worth keeping, though. I wouldn't fight removing it if nobody uses it)

Reply:
Ah that's a good point! I almost wonder if it's worth calling `cargo xtask test` for the CI, but obvious downside is each step wouldn't show up as nicely in the Actions UI :thinking: